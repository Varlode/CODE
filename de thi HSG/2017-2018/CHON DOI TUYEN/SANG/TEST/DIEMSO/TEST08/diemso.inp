1000
113963 64008 959664 795945 956675 448632 989649 863630 37349 703676 87189 385300 255098 87571 883844 870927 904862 1564 855897 199090 245989 211437 645915 808597 186266 417085 898256 701220 20446 539861 120358 855988 565664 535540 477934 480784 690213 869813 745631 388030 263361 787655 729424 308487 121715 360495 440780 223281 437861 597461 998214 508544 482573 793037 353408 561114 533954 245074 584766 324558 672418 698907 422236 402599 396409 139928 258568 67283 387777 819773 419986 611466 167940 710389 748781 52596 952917 809685 924213 522681 361262 759575 527270 973117 841670 628360 579048 266130 754316 387283 837559 252799 412728 554931 673517 559130 322683 708566 170284 742242 960375 442533 970651 14573 732399 630153 354830 896094 124437 123622 497240 572091 319368 110326 623609 846620 443626 912685 942723 274162 870149 230832 863365 272388 960355 870914 433467 542145 686629 500623 921902 52798 350133 685144 825398 445809 634119 189562 25711 726511 398072 406027 366809 167576 379174 722193 26937 488546 981168 647930 518010 612563 434008 47390 271188 348762 807889 270697 35541 186924 303350 952180 45309 590331 544053 206451 717638 130708 207863 853322 187375 306186 682803 146860 767073 755401 463376 760675 487309 964890 687630 310091 949602 802458 620587 226636 106231 657680 701887 857010 646409 187695 694149 222128 696708 966603 640793 288511 81968 702826 706922 330053 604784 439756 374713 498408 660371 769437 885917 97821 261070 495987 703346 810670 972150 869744 566618 973587 754832 906668 986029 465064 799364 385922 676554 260744 884795 223407 751078 328759 243689 102606 113360 310663 507589 744401 54092 272192 656515 189057 386222 231410 782890 185575 529237 845610 31712 550405 830962 553574 597584 953369 2114 698309 736024 260345 559494 42767 473127 27292 891358 793520 927253 550018 909490 607387 693633 978483 260334 512077 667990 871049 297580 207876 231815 256879 793651 589067 718654 916320 202091 785424 588130 791626 119448 683540 899441 76503 721050 103472 20649 34153 196842 71260 992927 291422 47683 221436 794679 252283 295536 932814 453511 84893 110925 549069 716653 630925 840816 382096 812979 636297 537551 27032 633484 787444 686090 778712 154308 86799 706478 536396 960308 320928 175679 616782 704839 118236 281825 768021 77205 553487 334484 640461 230315 465805 684463 739606 258886 321475 973211 51414 162285 972812 482120 18363 198029 948208 507856 960451 611534 85424 2975 27045 108941 274819 289341 992238 557778 490821 291883 837225 276770 305159 989465 331571 294783 378400 524444 424123 814828 616397 972038 255219 563244 125715 863676 143084 897759 325178 957506 85928 130123 299807 818788 407259 598732 208310 46422 221221 856098 554995 976557 42495 77720 16215 246397 205946 146600 8252 694569 766362 158009 62566 612122 693367 840708 980163 452680 277255 982805 408958 729266 341795 953825 282439 580679 561063 261071 855524 103797 223388 949707 36928 19004 473166 713594 831244 444683 632786 553173 859527 917226 928019 416948 3615 926137 941984 820444 837544 747485 921070 12495 966138 60740 856589 827346 65488 188755 689988 478029 342961 658248 696582 53273 50732 771391 42389 901062 747724 243288 753546 714899 878931 247877 815633 79950 747072 193234 769101 537083 658115 44076 860317 124799 584966 40298 145977 195149 514637 418453 430510 852344 167056 893037 662095 625094 373496 48911 925399 759776 100415 961584 484247 119867 58975 237699 423633 986378 726661 202561 762906 482632 293334 942374 737517 464850 798280 435102 89612 257951 787617 133207 882727 65844 903794 776971 924287 662514 434243 500545 757114 741365 501056 483239 847369 50971 882070 143948 910741 131879 60369 214292 85948 47821 29479 468957 152522 691723 869065 439069 304791 927050 965961 509237 377755 412142 580761 237326 960966 596263 313609 582078 718323 801481 730052 733517 576170 793340 271826 744869 429607 353079 57714 149228 614501 354539 968230 344764 795157 103138 729385 603080 586150 85700 603715 261795 130050 850799 643518 473625 72617 301529 229400 805241 648258 526082 604115 758388 933133 706816 958512 597043 552118 491450 134929 242497 92203 900905 592250 331817 231001 789266 131999 600022 388215 584420 882735 345941 949603 185759 593781 600810 964512 883629 620834 762196 902012 50259 459517 988933 501839 908102 231692 586852 614927 7762 315460 79297 277170 631081 674342 329487 149358 601330 802794 597595 222398 277516 692714 269144 367588 380909 214181 540634 458115 416956 190134 548192 807177 894652 261177 166501 193022 321222 642034 630568 153345 965498 136843 408100 64426 106117 348932 416961 414269 708782 914960 735321 584533 618507 614092 577453 268006 430320 418922 549986 712436 80894 177400 612754 789756 985022 694630 172936 635224 480161 385624 841924 862364 709275 400528 607466 77008 120780 369005 42546 751662 258717 433862 446401 953474 976244 456173 58908 114906 919959 185738 447019 324343 127288 168863 284523 943548 477414 603079 347183 723499 916290 147394 410323 793975 205622 783683 97892 236083 316296 456273 509178 307654 997924 322542 136737 542685 893014 552379 587401 738488 970671 298955 956519 148843 393138 694154 803354 255466 196356 213700 424500 271522 988646 175762 427859 755752 342067 63282 797963 3322 260085 775883 688485 384906 515833 537469 494710 571370 543770 826873 554224 961009 171715 807277 317768 287861 569894 195527 46353 263965 536911 256058 39706 809050 748622 573446 254449 540350 459776 524120 154140 484011 450860 964508 768615 562522 726091 961531 942341 388574 380723 544409 711070 169957 413096 729762 546528 357673 906986 557777 460861 421824 986864 716825 61557 367301 107742 358640 189160 721172 27565 67298 247748 165945 228404 413841 465675 851948 999278 133230 518106 833334 726532 654125 230948 606529 690355 21071 790458 776841 653814 605778 326743 411899 151309 277308 710207 201026 199377 575829 208292 389333 486384 904854 903871 686910 971470 214251 589432 410998 460587 693532 304816 830921 806951 531734 224180 415331 987168 988226 62758 179135 714350 561601 469078 549437 308902 735183 841469 608126 728714 528816 303114 568939 256192 92140 771319 29074 169202 801687 43034 858744 24539 291083 594810 851502 699031 868081 570555 259350 865244 727867 940397 628716 950899 153221 625461 802948 722739 879130 320543 351259 48077 10003 271666 35812 868134 450860 381836 490003 417306 713363 870434 95644 598869 540377 686051 348688 697873 637184 756534 574008 989636 887639 141719 681603 336265 937639 529326 688405 376368 485483 861345 784072 596661 459627 974291 519958 590991 828949 862707 321387 10147 633906 708773 132205 952427 947691 177133 454562 155298 909797 256089 578317 488567 172859 695642 515797 322984 195788 27259 978854 890498 15110 495014 803306 814086 468716 583584 662913 148076 438635 987447 572577 519113 201654 65456 691059 267851 572285 614671 433155 825510 833740 794010 599572 201702 163541 532037 599861 996809 660015 
3
34 188
45 158
69 184
