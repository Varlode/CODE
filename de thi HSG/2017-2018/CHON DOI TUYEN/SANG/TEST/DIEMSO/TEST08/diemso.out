79774272
58840867
61261083
